modules/_labs/badge/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,73 @@
+/// <reference path="../../../../../typings.d.ts" />
+import React from 'react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {Table} from '../../../../../utils/storybook';
+
+import Badge from '../index';
+import README from '../README.md';
+
+export default {
+  title: 'Labs|Badge/React/',
+  decorators: [withReadme(README)],
+  parameters: {
+    chromatic: {
+      disable: false,
+    },
+  },
+};
+
+export const VisualTesting = () => {
```

Comment:
If there's a better pattern for visual testing, please let me know. I went with a straightforward approach to describe all the permutations in a table format.

Reply:
I think this is our bad for not documenting better how we visual  test our components, at least our patterns. Instead of creating a new file just for visual test, we add another folder but just add the test within the same file. Look at `stories_Checkbox.tsx` as an example, specifically the `CheckboxStates` method.

Reply:
@mannycarrera4 with the move to the CSF format, this will have to be a separate file now that the parameters are defined in the default export (which is file-wide).

Honestly this is really going to be a pain when plugins can only be configured once for a whole file. cc @NicholasBoll

Reply:
ooo TIL